### ISSUE TITLE
Remove musician role

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#26323]
To be clear I'm fine with the gear existing, but with that being said musician serves no purpose.
- People that want to play music will just play music, ignoring any existing musicians that may exist. Same with forming bands.
- The drip is cool and can just be in a vendor.
- Mime has an actual integration of silence with the game, and clown has some special gear. You can't just pick up an instrument with these roles and invalidate the entire role existing in the first place unlike with musician.
- This is the other side of the coin of https://github.com/space-wizards/docs/pull/176

**Comment by SlamBamActionman** [comment]
I think that these arguments expand to a lot of roles that would be swept up in it. If that's good or bad, you decide.

Boxer, Psychologist and Zookeeper come to mind. You could even argue the Librarian (in its current state), as their department is nothing more than a glorified meeting room.

I could see Musician get a bit more value with loadouts in the future; start with your favorite instrument, instead of having to rely on Cargo. There's also some fun design space for job-specific uplink items.

**Comment by metalgearsloth** [comment]
> Boxer, Psychologist and Zookeeper come to mind. You could even argue the Librarian (in its current state), as their department is nothing more than a glorified meeting room.

They are inherently RP roles like Musician and fall under the same umbrella, but where I'd argue their different (apart from Boxer which I think is also dumb) is they can't just be ignored entirely by walking into the bar and playing an instrument.

**Comment by Hmeister-real** [comment]
I don't think that removing working and currently enjoyable content for absolutely no reason but "They serve no purpose at the moment, so delete them" is fair at all. Its like saying that non-doctors can heal the wounded, so dotors should be removed, but just like with surgery for doctors, i feel like the musician is still missing a key part of what makes or can make them good, like the Jukeboxes or something. Personally, I don't enjoy the idea of sitting in a place and listening to midis for a whole shift, but thats the musicians job, to spread music across the station and make the game more iconic, and people who play as one guarantee that happens, and at this point a silly musician in the background is honestly pretty iconic. This feels a bit long, but i just dont think that removing parts of the game for people for no reason but "functional pointlessness" is fair at all, and i don't think decisions like this should be discussed in the github, but on the discord publicly first before being suggested

**Comment by metalgearsloth** [comment]
> I don't think that removing working and currently enjoyable content

No content is being removed.

> i feel like the musician is still missing a key part of what makes or can make them good, like the Jukeboxes or something.

That just invalidates the role even harder.

> and at this point a silly musician in the background is honestly pretty iconic

Most of the time I see it's a passenger doing it, or maybe a random crewmember.

> This feels a bit long, but i just dont think that removing parts of the game...

This last sentence has no relation to anything.

**Comment by UbaserB** [comment]
Its a role because that’s what you do all shift. Greytiders don’t play music ALL of the shift, and it will be sad to be music-less

**Comment by EmoGarbage404** [comment]
> I don't think that removing working and currently enjoyable content for absolutely no reason but "They serve no purpose at the moment, so delete them" is fair at all. 

> but i just dont think that removing parts of the game for people for no reason but "functional pointlessness" is fair at all

that's not what he said so that's the first thing. i'm not sure you are aware what a quote is but don't phrase it like a quote if they literally did not say it.

> Its like saying that non-doctors can heal the wounded, so [SIC] dotors should be removed

MD as a role has a variety of functionality that make the role a valuable addition. Namely, a specific access for entering medbay and accessing medical equipment, a hierarchical structure within their department, and a general recognizable identity that helps others be aware of where they can receive treatment. 

People can treat themselves, but doctors are given the tools to be able to better treat others. All roles are inherently fluid and can be fulfilled by anyone but doctors are actually given leverage to do it better via structure and equipment.

> i feel like the musician is still missing a key part of what makes or can make them good, like the Jukeboxes or something.

Musicians have never received any kind of unique content ever since their inception. Even if you add jukeboxes--which i fail to see how they could enhance musician gameplay--i doubt it'd suddenly resolve all the earlier issues that don't really give them a larger purpose.

> but thats the musicians job, to spread music across the station 

it's not really a 'job' and the lack of scaffolding to accomplish this 'job' can frequently make it difficult and pointless. A bored tider can camp up on a piano in bar and play over you, leaving you with basically nothing to do. In fact, multiple people across the station can do exactly that. Your service extends as far as people not having access to instruments, which are probably some of the most plentiful fluff items in the game.

> make the game more iconic

debatable at best.

> i don't think decisions like this should be discussed in the github, but on the discord publicly first before being suggested

github is perfectly public and there's no issue discussing things here as opposed to discord.

**Comment by K-Dynamic** [comment]
Could make it so everyone who isn't a musician or knows how to play instruments plays MIDIs offkey :trollface: 

Then after 5 minutes of continous playing they know how to play it properly

**Comment by PJB3005** [comment]
> Boxer, Psychologist and Zookeeper come to mind. You could even argue the Librarian (in its current state), as their department is nothing more than a glorified meeting room.

* Psychologist is an actual real medical profession with responsibilities and privileges. It's also advertising to other people on the station "you can RP with me". At least ideally.
* Zookeeper I'm not familiar enough with, but it's station specific and at least the job name has a lot more IC implications than "anybody can do this".
* Librarian is a weak job that exists kinda more because it's grandfathered in from SS13 (where it has terrible gameplay, but has something). On some SS13 servers, Librarian also has an alt job title of being a reporter.
* Boxer shouldn't be a job in the first place either IMO.

Musician, Boxer, and artist are all "hobby" activities that don't require serious responsibilities or qualifications and can really be picked up by any crew member at any time. 

> Its a role because that’s what you do all shift. Greytiders don’t play music ALL of the shift, and it will be sad to be music-less

Passengers can literally just go and pick up the clothes, an instrument, and do exactly the same thing. They do not need a job title to be able to do it.

**Comment by Hanzdegloker** [comment]
I enjoy being able to have 2 things

A. a prefect excuse to be sitting in the bar for 2 hours playing songs

B. Guaranteed instruments round start to be able to play

Tis nice that your PDA can even be used as a instrument, if we had loadouts in which I could choose a trinket to have which is an instrument of choice I suppose I can get along with its removal but as is musician is my fav hands down I just want to vibe role to just exist in a server and rp alongside playing music 90% of the time

**Comment by PJB3005** [comment]
> A. a prefect excuse to be sitting in the bar for 2 hours playing songs

You don't need that. You can just join as passenger.

> B. Guaranteed instruments round start to be able to play

This is a mapping issue. And tbh if the issue is "3 other passengers already raided the mapped supply" then TBH take the L that round. Or buy instruments from cargo. Or scrounge through maint for them. It's fine.

**Comment by UbaserB** [comment]
I would think musicians would have a proper role on the station when they can make composite songs/DJ at a station so it requires proper skill and time/effort. This is going into the territory of ideaguys, but i feel that you guys only want to remove is because there’s not enough content to they point that they *feel* useless. 

Relate this to librarian; right now its a flavour role but as soon as shu’s rework is finished, they’re all of a sudden a real role.
Instead of removing, we should rework.

**Comment by PJB3005** [comment]
> I would think musicians would have a proper role on the station when they can make composite songs/DJ at a station so it requires proper skill and time/effort.

Ideaguysing a fancier instrument is not going to make their job any less replaceable. 

> Instead of removing, we should rework.

And if somebody can come up with a good design, then the job can be added. But I'm going to have my doubts for now.

**Comment by metalgearsloth** [comment]
> Relate this to librarian; right now its a flavour role but as soon as shu’s rework is finished, they’re all of a sudden a real role.
Instead of removing, we should rework.

To my memory we've always emphasised adding the feature then mapping it into the game and not mapping it into the game then maybe coming up with a feature later as we specifically didn't like how unitystation did it, back when they were further along than us.

**Comment by Just-a-Unity-Dev** [comment]
From the issue, what I understood was that musicians were pointless. Okay, and so is a majority of service? Clown, Mime, Lawyer, Librarian and the Zookeeper all have no mechanical features out of RP. Your argument of passengers should replace the musician doesn't make sense. Do you pay to board a luxury ship and then play the jazz, or do you listen to the band that's currently playing?

If you do go ahead with this (which you most likely will), then you should rename Passengers to Assistants, because now they're [actually assisting](https://github.com/space-wizards/space-station-14/pull/8007) with the station.

**Comment by metalgearsloth** [comment]
> From the issue, what I understood was that musicians were pointless. Okay, and so is a majority of service? Clown, Mime, Lawyer, Librarian and the Zookeeper all have no mechanical features out of RP. Your argument of passengers should replace the musician doesn't make sense. Do you pay to board a luxury ship and then play the jazz, or do you listen to the band that's currently playing?
> 
> If you do go ahead with this (which you most likely will), then you should rename Passengers to Assistants, because now they're [actually assisting](https://github.com/space-wizards/space-station-14/pull/8007) with the station.

What are you talking about

**Comment by Just-a-Unity-Dev** [comment]
> What are you talking about

tl;dr: we have several other jobs that might be considered pointless so why are you proposing to remove the musician and not the other roles

**Comment by Just-a-Unity-Dev** [comment]
ignore the jazz-music part i was dumb whilst writing that

**Comment by metalgearsloth** [comment]
> tl;dr: we have several other jobs that might be considered pointless

If you consider them pointless open issues for it.

**Comment by Hmeister-real** [comment]
yeah, why the hell are you and the other maintainers trying to entirely remove roles rather than reworking/improving them? Its not as if they are broken or widely hated, just a little stale since the boxer and musician are relatively unchanged since their original conception as a role

> this is gonna be a repeat of what happened with removing space pens or the removal of senior roles, isnt it? 😿
>>repost from boxer discussion since it also applies

**Comment by musicmanvr** [comment]
Why not just make "fluff" roles like the musician a flavor of the day type of thing? There's no need to remove a role entirely. Goon does this with their role of the day, where flavored roles are cycled per day. We don't necessarily need to force a set schedule with roles of the day, just have some of the lesser used roles cycle in and out.

**Comment by musicmanvr** [comment]
There's absolutely no reason to remove a role other than not knowing/understanding how to rework it to be unique enough or if it's completely useless.

**Comment by metalgearsloth** [comment]
> There's absolutely no reason to remove a role other than not knowing/understanding how to rework it to be unique enough or if it's completely useless.

waiting on suggestions.

**Comment by Just-a-Unity-Dev** [comment]
> waiting on suggestions.

could always have a mini-fl studio or a chrome music maker integrated with instruments, so that people could make songs whilst in game and not just downloading off the net. alongside that, you could always integrate a mini-rhythm game

these would make playing instruments much more fun rather than just play & afk

**Comment by PJB3005** [comment]
> From the issue, what I understood was that musicians were pointless. Okay, and so is a majority of service? Clown, Mime, Lawyer, Librarian and the Zookeeper all have no mechanical features out of RP.

I already explained half of these. Clown and mime are too much SS13 classic and have genuine mechanical differences too.

> Do you pay to board a luxury ship and then play the jazz, or do you listen to the band that's currently playing?

Passengers do not exist for an IC reason, they are solely there to solve the OOC factor that some players don't want serious responsibilities.

> could always have a mini-fl studio or a chrome music maker integrated with instruments, so that people could make songs whilst in game and not just downloading off the net. alongside that, you could always integrate a mini-rhythm game

Wow I countered this one too:

> Ideaguysing a fancier instrument is not going to make their job any less replaceable.

> Why not just make "fluff" roles like the musician a flavor of the day type of thing? There's no need to remove a role entirely. Goon does this with their role of the day, where flavored roles are cycled per day. We don't necessarily need to force a set schedule with roles of the day, just have some of the lesser used roles cycle in and out.

I think this is a fair compromise IMO, though I think maybe some general-purpose system to give people ideas what to do (like non-antag objectives) could be nice too instead.

**Comment by FairlySadPanda** [comment]
SUGGESTION IDEAGUY MOMENT

Roles like boxer, musician etc should be doled out by the HOP in-round rather than being roundstart.

The currently-useless Psychologist is just someone missing content atm rather than being innately useless.

**Comment by Everturning** [comment]
> I think the "everybody's happy" solution is to remove the actual musician job from prototypes, but keep "it" selectable in lobby (which gives you a passenger role but changes your job title on roundstart).

This doesn't make everyone happy, it's just a blanket statement.

Also, musician does have a purpose, and that's to play music. So what if other people can pick up instruments? That's good for them when they don't have a musician around, but most likely they'd stop playing music around an actual musician.

A musician has purpose, and that's to play music. It sounds nice for others, and it creates RP on occasion, and if it doesn't then that's fine. But that doesn't mean we should remove the role entirely just because it doesn't force people to talk to other people.

**Comment by superjj18** [comment]
I simply do not see the value in this.  You can argue that it doesn't remove anything critical, but I have yet to see any argument at all for this change adding to the enjoyment of the game.

you can say its an unimportant role and I will agree.  You can say anyone can do what this role can do, and ill agree aside from the unique PDA ability.  That being said you can't say removing the role adds any value to the game or fixes any existing issues, and for that reason I feel this is a needless and unpopular change to the game's current state, and for that reason I am against it.

**Comment by deathride58** [comment]
Honestly it feels like musicians would be *much* better off as an alt-title for passengers. In its current state, it's practically just a glorified alt-title anyhow, so transitioning musician to a proper alt-title (once someone actually makes an alt-title system) would come with all the benefits of removing musician (such as the decreased map maintenance overhead) without *actually* removing musician.

Since musician only has one slot, it's pretty common to see people just fall back to passenger, find an instrument roundstart, and get to jamming as though they've rolled musician regardless. Removing musician as a role wouldn't affect this behavior at all, as those who main musician are already more than used to acting as a musician when they're a passenger.

Adding alt-titles and making musician an alt-title for passenger would make it far easier for these players to signal "hey, I'm largely just here to chill and play some midis" without having the anxiety of stepping on the toes of whoever grabbed the musician slot.

Musician isn't the only role that'd benefit from alt-titles as a system being added to chargen; it'd also be viable to re-add senior roles but as job alt-titles for their respective jobs, for instance.

**Comment by moonheart08** [comment]
I don't think removing musician outright will really do anything beneficial.

While the job itself doesn't have much in the way of mechanics (something that imo could be fixed or improved on), jobs themselves have a purpose simply by existing, which is game-decided division of responsibility. When the game gives you a limited supply title like Musician, it's not just a fancy name (well, it *is*, but:), you're effectively being given the responsibility by the game and this has the extra effect of reducing how often other people will do the same thing. Which, in the case of music (or clowning, or boxing, or being librarian), is ultimately beneficial as it helps reduce the number of people vying to do some activity.

Essentially, think of these titles as a way to *discourage* people from all picking the same activity, and function as a sort of tiebreaker if multiple people want to do the same thing in the same space (The musician will likely be subconciously prioritized, because they have the fancy title and the passenger trying to play Never Gonna Give You Up doesn't.)

**Comment by moonheart08** [comment]
tldr: fancy clothes and fancy titles influence how people play, even if they don't have some coded in mechanical purpose.

**Comment by moonheart08** [comment]
> tldr: fancy clothes and fancy titles influence how people play, even if they don't have some coded in mechanical purpose. this same philosophy is why i okay'd boxer way back when, it's just a funny title and nothing else but that's enough to get people to act differently than they otherwise would when they get it.

I will note all of these jobs are part of an unfinished plan I had to introduce full-on wildcard/random roles the game throws in after round-start, to add a bit of variety to rounds by dropping in roleplay scenarios (traveling band, tourists, inspectors, etc) that players can choose to join in as.

**Comment by moonheart08** [comment]
@PJB3005 paging from the above comment as I know you mentioned dailies being a "fair compromise"

these jobs being dailies/wildcards was the original intent, it just never got implemented (but even without it I think they do good)

**Comment by wrexbe** [comment]
If you have the musician role, and someone is running around where you are working by playing music, I feel like it would be reasonable to escalate, or ask security to stop someone from disrupting your job. I would expect something similar to happen in real life.

**Comment by Killerqu00** [comment]
>     * People that want to play music will just play music, ignoring any existing musicians that may exist. Same with forming bands.
People that want to make drinks will just make drinks, there is even a maints bar for it.
People that want to play Librarian will just order books from Cargo and write books themselves.
People that want to play Clown will just ~~play on LRP and slip security~~ do silly pranks without having clown drip.
People that want to play Engineer will just get tools from ToolVendor.

Not every role should have something absolutely-unique-and-one-in-a-kind. Some roles do, like Mime and Cyborgs. But removing the role just because it can be done by other people is a bad decision. If anything, we should provide Musician with more gear, if that is a problem, not remove them entirely.

**Comment by IProduceWidgets** [comment]
The only reason I don't want this one removed is there is often a lot of "dead air" when playing the game, and having someone who has signed up specifically to spend the round filling said dead air is good for the ambiance. 

Could it be a loadout of passenger? Sure, but I wouldn't want it gone until that is a thing.

**Comment by FairlySadPanda** [comment]
I think there is scope for musician to have fun gameplay to be honest. They literally just need to have a lathe to make new instruments with. Means the synthesiser can be moved out of sci lathe.

But without a design doc it's... Yeah

**Comment by Jezithyr** [comment]
Honestly, Musician is an interesting concept for a role if only it had more content for it. In it's current state I sort of agree with PJB and Sloth that it doesn't fit as a separate job/role.  There is literally no difference between a musician and a passenger functionality, other than the round start instrument and stage access. 

That being said, I don't think that we should take the nuclear option of removing the role. Musician isn't creating any design or gameplay issues, it's simply just a role that doesn't have enough content or a design to justify it existing as a separate role. I really don't think we should be removing under-developed roles since that makes it much harder for contributors to create the role-based content that would justify the role in the first place. It's a chicken-vs-egg problem and I think we need to accept that we may have roles that aren't fully developed to allow contributors to add the content to justify that role.

Rather than removing the role, we should either A: do a design/expansion of musicians to add additional gameplay/objectives to make their role more unique (similar to clown). Or B: We should work on creating a system for passenger subroles, where players are randomly given objectives/special starting items (Or this subrole could be granted by HOP in round). In this case we would move musician to being a subrole under passenger.

Personally I feel like option B is the better option as it opens the door for many new "sub-roles" that while interesting, don't have the content to justify being their own job/role. Additionally, I think that instead of just suddenly dropping a PR that removes a role from the game we should instead be encouraging contribs to expand on the role and update it's design. Only if that doesn't work and the role is creating unhealthy gameplay should we remove it.

**Comment by moonheart08** [comment]
> Honestly, Musician is an interesting concept for a role if only it had more content for it. In it's current state I sort of agree with PJB and Sloth that it doesn't fit as a separate job/role. There is literally no difference between a musician and a passenger functionality, other than the round start instrument and stage access.
> 
> That being said, I don't think that we should take the nuclear option of removing the role. Musician isn't creating any design or gameplay issues, it's simply just a role that doesn't have enough content or a design to justify it existing as a separate role. I really don't think we should be removing under-developed roles since that makes it much harder for contributors to create the role-based content that would justify the role in the first place. It's a chicken-vs-egg problem and I think we need to accept that we may have roles that aren't fully developed to allow contributors to add the content to justify that role.
> 
> Rather than removing the role, we should either A: do a design/expansion of musicians to add additional gameplay/objectives to make their role more unique (similar to clown). Or B: We should work on creating a system for passenger subroles, where players are randomly given objectives/special starting items (Or this subrole could be granted by HOP in round). In this case we would move musician to being a subrole under passenger.
> 
> Personally I feel like option B is the better option as it opens the door for many new "sub-roles" that while interesting, don't have the content to justify being their own job/role. Additionally, I think that instead of just suddenly dropping a PR that removes a role from the game we should instead be encouraging contribs to expand on the role and update it's design. Only if that doesn't work and the role is creating unhealthy gameplay should we remove it.

I don't think we need some new job system for this. Jobs are just a title and equipment, what you're describing doesn't actually change anything with how they ultimately work. "subroles" aren't a meaningful concept here (imo) when roles themselves are already flexible enough without them.

We should be asking what it means for something to be a role, because when I programmed the job system all that roles were to me is a loadout and fancy title. Proper *departmental jobs* should have the extra weight, yes, but Musician and the other civ roles are explicitly in the "not really a department, low stakes" department already.

**Comment by FairlySadPanda** [comment]
# Spitballs
The musician should be able to get a super synth somehow (it's the best MIDI player in the game because it's the only one that is multi-sample)
The musician should be able to craft instruments
The musician should be able to broadcast music somehow so people can tune in, because "radio host" etc roles are literally always rubbish
The musician should probably have something fun for tot work...

**Comment by moonheart08** [comment]
> # Spitballs
> 
> The musician should be able to get a super synth somehow (it's the best MIDI player in the game because it's the only one that is multi-sample) The musician should be able to craft instruments The musician should be able to broadcast music somehow so people can tune in, because "radio host" etc roles are literally always rubbish The musician should probably have something fun for tot work...

Supersynth is never to be made obtainable, it's a debug item. They can make a DAW using crafting like everyone else.

**Comment by FairlySadPanda** [comment]
> Supersynth is never to be made obtainable, it's a debug item. They can make a DAW using crafting like everyone else.
 
By "DAW" you mean "drag around five pianos behind them and inexplicably play them all"? ;)

A "really good instrument" as a round goal for a musician would be the goal, it doesn't need to be the current supersynth.

**Comment by Killerqu00** [comment]
> > Honestly, Musician is an interesting concept for a role if only it had more content for it. In it's current state I sort of agree with PJB and Sloth that it doesn't fit as a separate job/role. There is literally no difference between a musician and a passenger functionality, other than the round start instrument and stage access.
> > That being said, I don't think that we should take the nuclear option of removing the role. Musician isn't creating any design or gameplay issues, it's simply just a role that doesn't have enough content or a design to justify it existing as a separate role. I really don't think we should be removing under-developed roles since that makes it much harder for contributors to create the role-based content that would justify the role in the first place. It's a chicken-vs-egg problem and I think we need to accept that we may have roles that aren't fully developed to allow contributors to add the content to justify that role.
> > Rather than removing the role, we should either A: do a design/expansion of musicians to add additional gameplay/objectives to make their role more unique (similar to clown). Or B: We should work on creating a system for passenger subroles, where players are randomly given objectives/special starting items (Or this subrole could be granted by HOP in round). In this case we would move musician to being a subrole under passenger.
> > Personally I feel like option B is the better option as it opens the door for many new "sub-roles" that while interesting, don't have the content to justify being their own job/role. Additionally, I think that instead of just suddenly dropping a PR that removes a role from the game we should instead be encouraging contribs to expand on the role and update it's design. Only if that doesn't work and the role is creating unhealthy gameplay should we remove it.
> 
> I don't think we need some new job system for this. Jobs are just a title and equipment, what you're describing doesn't actually change anything with how they ultimately work. "subroles" aren't a meaningful concept here (imo) when roles themselves are already flexible enough without them.
> 
> We should be asking what it means for something to be a role, because when I programmed the job system all that roles were to me is a loadout and fancy title. Proper _departmental jobs_ should have the extra weight, yes, but Musician and the other civ roles are explicitly in the "not really a department, low stakes" department already.
> 
> I don't think giving them the further distinction of "subrole" actually does anything except appease some folks who misunderstand how much weight a job entry has.

There are tons of RP roles in the game: clown, mime, librarian, psychologist, chaplain... Should we remove all of these as well?

**Comment by Killerqu00** [comment]
> > clown, mime, librarian, psychologist, chaplain
> 
> Psychologist is de-facto RP-flavored medical doctor, agree with you on that. So if musician is passenger with id change (+ starting gear), then psychologist is a medical doctor with id change (+ starting gear).
> 
> Chaplain has a roadmap with vampire antags, revenant interactions, and generally all things holy and unholy. Who's going to protect the station when supernatural disasters strike?
> 
> Clown and mime are kept because they are part of SS14 identity[1](#user-content-fn-1-93c8349e02af44befc554f528d831d31).
> 
> As for librarian - there's been hushed whispers of removing that role too, but they quietened now because there have been several suggestions and steps that could give the role a strong station identity.
> 
> Basically it comes down to:
> 
>     * If role has a roadmap/design/fulfills a hard role on a station, whether currently or upcoming = keep it as a separate role.
> 
>     * If it's just RP-flavor on top of existing role (psychologist/musician/boxer) = we are discussing what we do about it in this issue.
> 
>     * Clown and mime get special treatment = self-explanatory.
> 
> 
> ## Footnotes
> 
>     1. And I feel there is a little more than that going on here. A musician is a passenger that plays music. A psychologist is a medical doctor that specializes in psychology. What is a clown to a passenger? What is a service worker to a mime? They don't compare. They aren't just RP-flavored _some-other-role_. They are distinct. They are fully blown RP-roles, not just flavors. [↩](#user-content-fnref-1-93c8349e02af44befc554f528d831d31)

I would argue that musician is also a part of SS14 identity. SS14 is not SS14 if a musician does not blast absolute bangers around the station in MIDI format. This definitely deserves a role.

**Comment by potato1234x** [comment]
i personally sometimes just want to sit on my ass all round and play music and have access to pretty much all instruments

**Comment by GNF54** [comment]
Musician's have PDAs that can play Midis

**Comment by metalgearsloth** [comment]
Now that https://github.com/space-wizards/space-station-14/pull/25715 is merged just put it on the passenger role so as many people can be musician as they want as I mentioned.

**Comment by Froozigiusz** [comment]
>Now that https://github.com/space-wizards/space-station-14/pull/25715 is merged just put it on the passenger role so as many people can be musician as they want as I mentioned.

If you'll really remove an entire role (much different in it's gimmick from a lowly passengr, mind you) from the game „just 'cause you feel like it”, then at least give us a way to modify/pick a job title within load–out section. I wouldn't want to see entire swarms of grey–shirts just to get a proper title.

**Comment by Cadunkus** [comment]
No this is a good change. You can still be a musician, just pick passenger and go play music the entire round. That's how greyshirt gimmicks work.

Cordoning what should be a public recreational activity into its own job is pointless. It's like if exercise machines were common on space stations because mappers liked adding them and some rando decided "Hey there should be a weightlifter job where they have no access or mechanics and all they do all shift is spam the exercise machines and get upset whenever someone else decides to use those!"

**Comment by Djungelskog2** [comment]
having it be one-of-a-kind each round and them having stage and backstage access is what makes it special, along with the outfit and PDA just like basically every other job.
Any role could be played by a greytider if they go to the HoP and pick up a few tools thats the POINT.

every point in removing the musician could be the same for other roles, like the clown for example, Theatre access, easily replaced by a passenger with theatre access where they just pick up some pies and banana peels and start acting, the only truly mechanical difference is the hard-coded clumsiness mechanic

**Comment by Cadunkus** [comment]
> ...any role could be played by a greytider if they go to the HoP and pick up a few tools...

That's a strawman argument, don't do that... You don't jump counters to steal jobs for several reasons and that's actually why the musician _job_ should step aside so the musician _role_ can be public: So you're not intruding on the existing musician's "turf" by daring to play an instrument as a passenger.

In servers without the musician being cordoned off into a job slot, MIDI music _thrives_ because "Anyone can cook" -The fatass from Ratatouille

Heck [here's a MIDI library](https://www.dropbox.com/scl/fo/ds31jkzphf2uvhyqvkjc8/ACobcWtFkppNDLGIyerg5u4?rlkey=qxi8fcoka3hjyuox8xhsheir3&e=1&dl=0) someone made for Space Station 13. No musician job there.

**Comment by Djungelskog2** [comment]
> > ...any role could be played by a greytider if they go to the HoP and pick up a few tools...
> 
> That's a strawman argument, don't do that... You don't jump counters to steal jobs for several reasons and that's actually why the musician _job_ should step aside so the musician _role_ can be public: So you're not intruding on the existing musician's "turf" by daring to play an instrument as a passenger.
> 
> In servers without the musician being cordoned off into a job slot, MIDI music _thrives_ because "Anyone can cook" -The fatass from Ratatouille
> 
> Heck [here's a MIDI library](https://www.dropbox.com/scl/fo/ds31jkzphf2uvhyqvkjc8/ACobcWtFkppNDLGIyerg5u4?rlkey=qxi8fcoka3hjyuox8xhsheir3&e=1&dl=0) someone made for Space Station 13. No musician job there.

Why should the musician be removed and not roles like the clown, mime, chaplain and friends
musicians probably aren't going to come up to you to tell you to piss off for playing music, you could join in, go do your own thing or even also become a musician via the HoP. These standards your saying exist do not, these roles just provide more opportunity for those who want it, there is nothing preventing you in the round from doing the same.

**Comment by Djungelskog2** [comment]
Also yes musician exists in 13??? What the fuck are you talking about

**Comment by Cadunkus** [comment]
Okay @Djungelskog2 you like when people play MIDIs on the instruments. I like when people play MIDIs on the instruments. It adds to the atmosphere and all that jazz.

Musician job exists? Only one person plays MIDIs on the instruments.

Musician _role_ exists? Several people of different already active jobs plays MIDIs on the instruments.

Being a maestro all round started as a fun gimmick. Ironically, adding onto that gimmick by making it a job with an ID only restrained it. Trust me, the musician scene would thrive without the ID and I'm not speculating I've seen it in every station server where musician isn't a job. I would like that to be brought back.

**Comment by Djungelskog2** [comment]
This is an invincible ignorance fallacy...
Like I said before, the musician job is t and hasn't stopped anyone from ever playing music and it never will, people do, have, and will continue to do so. Have you considered that the servers you have been in just didn't have players who wanted to play midis? What makes you believe this connection even exists?

**Comment by Cadunkus** [comment]
The rounds in servers where musician wasn't restrained to a job did have several players who want to play MIDIs because there were several players doing just that. The rounds in servers where musician was restrained to a job rarely had more than one.

Musician should be a role, not a job, and I've gone over several reasons why. I have yet to see reasons why it should remain confined to such.

**Comment by Djungelskog2** [comment]
And here I am saying your wrong, your reasons are substantial and I have yet to see reasons why it should be changed.

**Comment by Cadunkus** [comment]
> And here I am saying your wrong, your reasons are substantial and I have yet to see reasons why it should be changed.

Glad I could bring you around to my point of view! :)

**Comment by ThatGuyUSA** [comment]
Necromancy!!

Would like to mention that boxer and zookeeper were removed for similar reasons mentioned here, and yet the musician remains.

**Comment by ProPeperos** [comment]
At this point we could remove clown and mime as well, as they are pretty useless roles too :godo:

but seriously
zookeeper was a station specific role (oasis station), it didn't have any special interactions with anything, roleplay boiled down to "look at animals"
boxer, similarly didn't have much roleplay opportunities and the interactions boiled down to "go and fight with me" which it wasn't very appealing

musician on the other hand is an entertainer, just like mime and clown, but instead of entertaining with shananigans you entertain with music and that's your main goal to share it, it's quite a good RP experience as people like listening to music, you can even get requests, talk about how cool the music is or if you recognise it. 
Just because a passenger can also play instruments, doesn't mean they going to stick to it for the entire shift, and having it a job means that people are dedicated to play music for the entire shift, not just on a whim

**Comment by ThatGuyUSA** [comment]
i just wanna close the issue tbh, this is more of a forum post 

number go down and all

**Comment by ProPeperos** [comment]
ye, i don't see anyone complain about it nowadays, could be reopened again if it is an actual issue
